homeassistant/components/keba/services.yaml

```diff
@@ -38,6 +38,21 @@ disable:
   description: >
     Stops the charging process if charging station is authorized.
 
+set_text:
```

Comment:
Some comment as above, sending the message is done via notify not via a service call anymore.